index.js

```diff
@@ -193,6 +193,25 @@ exports.format = function(name, fmt){
 };
 
 /**
+ * Returns formatted string. Useful when using a function as a format.
+ *
+ * @param {object} req
+ * @param {object} res
+ * @param {string} format
+ * @returns {string}
+ */
+exports.getFormatted = function(req, res, format) {
+  if(!req['headers']){
+    deprecate('getFormatted expects a req and request object as parameters.')
```

Comment:
A new function would not have deprecated behavior. If this is to catch a mistake, it should throw instead.